### ISSUE TITLE
Clarify which (deduped) hearings list will apply to project's dispositions and which apply to assignment dispositions

### ISSUE THREAD

**Issue by godfreyyeung** [NYCPlanning/labs-zap-search#815]
We need to decide whether the hearings lists in each of the following places should show ALL hearings in a project, or show only the assignment hearings.  

- show-project  (project profile) page "To Review section"
- show-project  (project profile) page milestones listing
- upcoming-project-card
- to-review-project-card
- reviewed-project-card
- hearing form confirmation modal
- recommendation form quorum question
- recommendation form confirmation modal

Basically,  this will boil down to either passing `project.dispositions` or `assignment.dispositions` into components like `deduped-hearings-list

**Comment by trbmcginnis** [comment]
Here's what I'm thinking:

*On any milestones list that's shown on tabs or on `show-project.js`:
- only show hearings that are associated with that project's milestone
- do this through `project-milestone` component, use lookup table to compare dispositions and milestones and create a new array of dispositions that ONLY has the dispositions associated with that project's milestone

*On Recommendation and Hearing Form
- only show hearings that are associated with that user's current project --> `assignment.dispositions`

*`to-review` tab:
- this is where we need design help --> should we only show the hearings that a user just submitted for that project OR should we show ALL hearings associated with that project that have been submitted
- if show ALL hearings associated with a project, then denote which user submitted each hearing (so it would be `project.dispositions`)

**Comment by godfreyyeung** [comment]
The design decisions on this may also affect whether hearings computed are moved from the project model to the assignments model, or if both models will have the hearings computed.

**Comment by godfreyyeung** [comment]
show-project (project profile) page "To Review section" -- assignment.disposition
show-project (project profile) page milestones listing -- project.disposition . 
upcoming-project-card -- project.dispositions
to-review-project-card .-- assignment.dispositions
reviewed-project-card -- project.dispositions
hearing form confirmation modal -- assignment.disposition
recommendation form quorum question -- assignment.disposition 
recommendation form confirmation modal -- assignment.disposition
